charts/scalardl-audit/templates/auditor/deployment.yaml

```diff
@@ -41,25 +41,30 @@ spec:
         - name: scalardl-auditor-properties-volume
           configMap:
             name: {{ include "scalardl-audit.fullname" . }}-auditor-properties
-      {{- if .Values.auditor.tls.caRootCertSecret }}
-        - name: scalardl-auditor-tls-ca-root-volume
+      {{- if and .Values.auditor.tls.enabled .Values.auditor.tls.certManager.enabled }}
+        - name: scalardl-auditor-tls-volume
           secret:
-            secretName: {{ .Values.auditor.tls.caRootCertSecret }}
+            secretName: {{ include "scalardl-audit.fullname" . }}-tls-cert
       {{- end }}
-      {{- if .Values.auditor.tls.certChainSecret }}
-        - name: scalardl-auditor-tls-cert-chain-volume
-          secret:
-            secretName: {{ .Values.auditor.tls.certChainSecret }}
-      {{- end }}
-      {{- if .Values.auditor.tls.privateKeySecret }}
-        - name: scalardl-auditor-tls-private-key-volume
-          secret:
-            secretName: {{ .Values.auditor.tls.privateKeySecret }}
+      {{- if and (.Values.auditor.tls.enabled) (not .Values.auditor.tls.certManager.enabled) }}
+        - name: scalardl-auditor-tls-volume
+          projected:
+            sources:
+              - secret:
+                  name: {{ .Values.auditor.tls.caRootCertSecret }}
+              - secret:
+                  name: {{ .Values.auditor.tls.certChainSecret }}
+              - secret:
+                  name: {{ .Values.auditor.tls.privateKeySecret }}
       {{- end }}
-      {{- if .Values.auditor.tls.caRootCertForLedgerSecret }}
-        - name: scalardl-auditor-tls-ca-root-for-ledger-volume
+      {{- if .Values.auditor.tls.enabled }}
+        - name: scalardl-auditor-tls-for-ledger-volume
           secret:
+            {{- if .Values.auditor.tls.caRootCertForLedgerSecret }}
             secretName: {{ .Values.auditor.tls.caRootCertForLedgerSecret }}
+            {{- else }}
+            secretName: {{ include "scalardl-audit.fullname" . }}-tls-cert
+            {{- end }}
```

Comment:
If you set `auditor.tls.caRootCertForLedgerSecret` explicitly, this chart uses the specified root CA certificate. However, if you don't specify that parameter, this chart uses the same root CA certificate as TLS of ScalarDL Auditor.

In other words, if you use the same CA to issue certificates for ScalarDL Ledger and ScalarDL Auditor, you don't need to set the root CA certificate manually.

You can see the conditions on the document side.
https://github.com/scalar-labs/docs-internal-orchestration/pull/24/files#r1584188835